src/Aspire.Hosting/ParameterResourceBuilderExtensions.cs

```diff
@@ -165,12 +166,20 @@ internal static IResourceBuilder<T> AddParameter<T>(this IDistributedApplication
             IsHidden = true,
             Properties = [
                 new("parameter.secret", resource.Secret.ToString()),
-                new("parameter.value", resource.Value) { IsSensitive = resource.Secret },
                 new(CustomResourceKnownProperties.Source, resource.ConfigurationKey)
             ],
             State = KnownResourceStates.Running
         };
 
+        builder.Eventing.Subscribe<AfterResourcesCreatedEvent>(async (evt, ct) =>
```

Comment:
Not sure we want to use this one,

Reply:
Ah, `ProcessResourceWithoutLifetime` didn't show up when I started writing my comment, I'll take a look at that.